### ISSUE TITLE
Support bindings to camelcased SVG attributes

### ISSUE THREAD

**Issue by wesleycho** [angular/angular#1159]
Certain svg attributes are camelcased, such as `viewBox`. However, `[viewBox]` would be recognized by browsers as `[viewbox]` since it is not a standard SVG attribute, and so the important camelcasing information is lost.

There should be a convention in Angular 2 for supporting bindings to SVG attributes.

**Comment by mhevery** [comment]
Does `[view-box]` not work?

**Comment by pkozlowski-opensource** [comment]
Hmm, I think we've got a non-addressed problem here. _Normally_ we could use `[attr.view-box]="expr"`  to bind to an attribute. But currently we are not doing dash->camel case conversion in a compiler since there are attributes (ex.: `aria-role`) where a dash a valid part of an attribute name... 

So the crux of a problem is that we've got both camel and dash-cased valid attribute names....

As for a solution I can think of:
- special-case svg attributes (ex.: `[svg.view-box]="expr"`) and do dash->camel case conversion for those
- special-case aria attributes (we used to have this in a compiler) 

I would probably "vote" for special-casing SVG attributes since those are the only ones (?) for which browser will allow camel-cased attributes.
